statistics.rb

```diff
@@ -0,0 +1,11 @@
+def top_reader
+    orders_grouped = orders.group_by(&:reader)
+    tab = orders_grouped.max_by { |_entity, reader| reader.count }.first
+    puts "#{tab} is the top reader"
+  end
+
+  def top_book
+    orders_grouped = orders.group_by(&:book)
+    tab = orders_grouped.max_by { |_entity, book| book.count }.first
+    puts "#{tab} is the top book"
+  end
```

Comment:
this should be module, not just randomly thrown methods into a file.
As I stated previously, that would let you extend your library class and call this methods right on it
```
lib = Library.new
lib.create_data
lib.top_reader
lib.top_book
```

Reply:
+

Reply:
added module for my methods(top_reader and top_book) and called them in main.rb